tictactoe/game2.rb

```diff
@@ -25,36 +28,31 @@
 
 while turn > 0
   puts
-  puts grid
+  grid_rows.each do |row|
+    puts row
+  end
   puts
 
-  input = $stdin.gets[0]
-  turn = turn-1
+  input = $stdin.gets[0].to_i
 
+  # now that we have input from 1..9,
+  # calculate which row and column 
+  row_number = (input-1) / 3
+  column_number = (input-1) % 3
```

Comment:
Why are we calculating which row and column? What does "/" and "%" translate into, in English? And why -1 from the input? Might be missing something super obvious